src/api_data.js

```diff
@@ -1,5 +1,5 @@
-export function gotData(url){
-	return fetch(url, {mode: 'cors'})
+export  function gotData(url) {
+  return fetch(url, {mode: 'cors'})
     .then(function(response) {
```

Comment:
lets use async/await or promises

Reply:
I am shocked for this review, the function gotData use then() to wait until the promise is resolved.